ClientApp/viking-vault/src/AttachCardModal.tsx

```diff
@@ -0,0 +1,215 @@
+import React from "react";
+import { constants } from "./ConstantVariables";
+import './AttachCardModal.css';
+
+const baseUrl = constants.baseUrl;
+var regex = /^([+-]?[1-9]\d*|0)$/;
```

Comment:
Please avoid using `var`s in JS as it's an older kind of declaring an variable, use `let` and `const` instead. (they behave differently and you should only use `var` if you want the functionality that the var provides or if you need to support an older browser and you don't transpile your code for some reason